FirebaseSessions/Tests/TestApp/Podfile.lock

```diff
@@ -0,0 +1,79 @@
+PODS:
```

Comment:
Is this file meant to be added? Otherwise the rest of the code LGTM!

Reply:
Podfile.lock is the file that contains the version of the dependencies that is used for referencing dependencies and their version number. Without this, Cocoapods will continue to use the older version of the dependency. Since, we are updating the version of GoogleUtilities, I chose to add this file in to say that we work only with a particular version of GoogleUtilities and above.

Reply:
Sounds good!